pkg/controller/nodelifecycle/node_lifecycle_controller_test.go

```diff
@@ -785,13 +796,13 @@ func TestMonitorNodeHealth(t *testing.T) {
 
 	for testName, tt := range tests {
 		t.Run(testName, func(t *testing.T) {
-			_, ctx := ktesting.NewTestContext(t)
+			tCtx := ktesting.Init(t)
```

Comment:
@pohly which context is the right one to use? I am lost between available contexts for tests.

Reply:
Both are functionally equivalent as far as contextual logging goes. tCtx offers some additional APIs, but that's a bit experimental.